integration/mock-backend-server/src/main/java/org/wso2/choreo/connect/mockbackend/MockBackendProd.java

```diff
@@ -293,6 +293,9 @@ public void configure(HttpsParameters params) {
             httpServer.createContext(context + "/echo", Utils::echo);
             httpServer.createContext(context + "/echo2", Utils::echo);
 
+            // below context is used to test response body passing test case
+            httpServer.createContext(context + "/requestBodyPass", Utils::echo);
```

Comment:
can't we use the existing `/echo` endpoint

Reply:
Check with `/echo` context.